code/Modules/Items/Carriables/Item.cs

```diff
@@ -67,17 +68,11 @@ public override string ToString()
             return $"[{Data.Id}] {Data.Name}";
         }
 
-        protected bool Equals(Item other)
-        {
-            return Data.Id == other.Data.Id;
-        }
-
         public override bool Equals(object obj)
         {
             if (ReferenceEquals(null, obj)) return false;
             if (ReferenceEquals(this, obj)) return true;
-            if (obj.GetType() != GetType()) return false;
-            return Equals((Item)obj);
+            return false;
         }
```

Comment:
Makes the equals method useless.
Why want reference equality when you can use == for references ?

Reply:
Yeah but we need to find another method for equals
The client doesn't have the data yet when the sbox method OnSceneAdd (or something) is called clientside